recipe_gdal.Dockerfile

```
@@ -149,20 +150,23 @@ RUN yum install -y \
     yum clean all
 
 # install
-RUN TEMP_DIR=/tmp/tiff; \
-    mkdir -p "${TEMP_DIR}"; \
+RUN TEMP_DIR="/tmp${STAGING_DIR}"; \
+    REPORT_DIR="${STAGING_DIR}/usr/local/recipe"; \
```

```suggestion
    REPORT_DIR="${STAGING_DIR}/usr/local/share/just/info"; \
```

Or is changing that dir everywhere a lot harder?